plugins/stupid.py

```diff
@@ -16,390 +16,389 @@
+#                     chan, nick, fine, totalfines[1:])
+#         else:
+#             if randnum == 1:
+#                 out = citation(db, chan, thing, 'shill {}'.format(nick))
+#             elif randnum == 2:
+#                 out = citation(db, chan, thing, 'shill {}'.format(nick))
+#             elif randnum == 3:
+#                 out = citation(db, chan, thing, 'shill {}'.format(nick))
+#             else:
+#                 fine = float(random.randint(1500, 3000))
+#                 nick = input.nick
+#                 try:
+#                     totalfines = float(database.get(db, 'users', 'fines', 'nick', nick)) + fine
+#                 except:
+#                     totalfines = 0.0 + fine
+#                 database.set(db, 'users', 'fines', totalfines, 'nick', nick)
+#                 fine = "{:,}".format(fine)
+#                 totalfines = "{:,}".format(totalfines)
+#                 if totalfines[0] == "-":
+#                     out = u"PRIVMSG {} :\x01ACTION fines {} \x02${}\x02 for breaking the NDA. You have: \x0309${}\x02\x01".format(
+#                         chan, nick, fine, totalfines[1:])
+#                 else:
+#                     out = u"PRIVMSG {} :\x01ACTION fines {} \x02${}\x02 for breaking the NDA. You owe: \x0304${}\x02\x01".format(
+#                         chan, nick, fine, totalfines)
+#     elif command == 'buy':
+#         randnum = random.randint(1, 4)
+#         if len(inp) == 0:
+#             if random.randint(1, 3) == 2:
+#                 out = citation(db, chan, thing, 'buy {}'.format(nick))
+#             else:
+#                 out = u"PRIVMSG {} :\x01ACTION pisses on the floor and leaves.\x01".format(chan)
+#         else:
+#             if randnum == 1:
+#                 out = citation(db, chan, thing, 'buy {}'.format(nick))
+#             elif randnum == 2:
+#                 out = citation(db, chan, thing, 'buy {}'.format(nick))
+#             elif randnum == 3:
+#                 out = citation(db, chan, thing, 'buy {}'.format(nick))
+#             else:
+#                 out = u"PRIVMSG {} :\x01ACTION pisses on the floor and leaves.\x01".format(chan)
+#     else:
+#         if len(inp) == 0:
+#             if random.randint(1, 3) == 2:
+#                 out = citation(db, chan, nick, "for {}".format(actions[command][1]))
+#             else:
+#                 out = u"PRIVMSG {} :\x01ACTION {}s {}\x01".format(chan, command, nick)
+#         else:
+#             randnum = random.randint(1, 4)
+#             if randnum == 1:
+#                 out = citation(db, chan, nick, "for {}".format(actions[command][1]))
+#             elif randnum == 2:
+#                 out = citation(db, chan, target,
+#                                "for being too lewd and getting {}".format(actions[command][0]))
+#             else:
+#                 out = u"PRIVMSG {} :\x01ACTION {}s {}\x01".format(chan, command, target)
+#     conn.send(out)
```

Comment:
One of the main advantages of git is the fact that removed code can be accessed after it has been removed..